test/e2e-tests/README.md

```diff
@@ -16,17 +16,33 @@ The following enviroment varaibles are required to run the test setup and the te
 - `S3_ENDPOINT` - Endpint of the bucket
 - `IMAGE_REGISTRY_USERNAME` - quay.io username
 - `IMAGE_REGISTRY_PASSWORD` - quay.io password
-- `S3_BUCKET` - Name of S3 bucket that has the model
-- `TARGET_IMAGE_TAGS_JSON` - JSON array of image tags that the final image will be pushed to. E.g. '["quay.io/user/model-name:e2e-test"]'
-- `NAMESPACE` - Cluster namespace used for testing
+
+The following enviroment varaibles are optional. You may still need to set them for the tests to pass depending on your setup. Read [here](../../pipelines/README.md#ai-edge-end-to-end-pipeline) for more context on how to set these up.
+- `SELF_SIGNED_CERT` - Path self signed cert to be used in pipeline
+
+
+```bash
+make go-test-setup
+```
 
 ## Run tests locally
+
+The following enviroment varaibles are required to run the tests. If any of these are not set then the tests will not run. Read [here](../../pipelines/README.md#ai-edge-end-to-end-pipeline) for more context on how to set these up.
+
+- `S3_BUCKET`	- Name of S3 bucket that contains the models
+- `NAMESPACE`	- Cluster namespace that tests are run in
+- `TARGET_IMAGE_TAGS_JSON`	- JSON array of image tags that the final image will be pushed to. E.g. '["quay.io/user/model-name:e2e-test"]'
+
+The following enviroment varaibles are optional. You may still need to set them for the tests to pass depending on your setup. Read [here](../../pipelines/README.md#ai-edge-end-to-end-pipeline) for more context on how to set these up.
+- `SELF_SIGNED_CERT` - Path self signed cert to be used in pipeline
```

Comment:
See above about clarifying that this is a local file on the user's machine

Reply:
👍